docs/tutorial/force_calibration.rst

```diff
@@ -464,3 +464,102 @@ When working with small beads or at high laser powers, it is important to verify
+
+Automated spurious peak detection
+---------------------------------
+
+We will continue the tutorial with the results of the previous section. If you did not yet do that part of the tutorial, please go back and execute the code examples in that section. 
+
+We still have the power spectrum `ps` that was created without blocking or windowing. Here we will use it to identify the peak and automatically obtain frequency exclusion ranges. We will use the method :meth:`~lumicks.pylake.force_calibration.power_spectrum.PowerSpectrum.identify_peaks()` in order to do so. This method takes a function that accurately models the power spectrum as a function of frequency, in order to normalize it. It then identifies peaks based on the likelihood of encountering a peak of a certain magnitude in the resulting data set. If we have a "good fit", then the easiest way to get that function is to extract it from our fit parameters and the model that was used during fitting::
+
+    params = [v.value for k, v in fit.results.items() if k in ['fc', 'D', 'f_diode', 'alpha']]
+    model_fun = lambda f: fit.model(f, *params)
+
+If there are no spurious peaks, then normalizing the unblocked power spectrum results in random numbers with an exponential distribution with a mean value of 1. The chance of encountering increasingly larger numbers decays exponentially, and this fact is used by `identify_peaks()`::
+
+    frequency_exclusions = ps.identify_peaks(model_fun, peak_cutoff=20, baseline=1)
+
+The parameter `peak_cutoff` is taken as the minimum magnitude of any value in the normalized power spectrum in order to be concidered a peak. The default value is 20, and it corresponds to a chance of about 2 in a billion of a peak of magnitude 20 or larger occuring naturally in a data set. If a peak is found with this or a higher magnitude, the algorithm then expands the range to the left and right until the first time the power spectrum drops below the value `baseline`. The frequencies at which this occurs end up as the lower and higher frequency of a frequency exclusion range. As such, the value of `baseline` controls the width of the frequency exclusion range. We can visualize the excluded peaks as follows::
+
+    fig, ax = plt.subplots(1, 2, sharey=True)
+    for axis, title in zip(ax, ('Full spectrum', 'Zoom')):
+        axis.loglog(ps.frequency, ps.power, label="Power spectrum")
+        for idx, item in enumerate(frequency_exclusions, 1):
+            to_plot = np.logical_and(item[0] <= ps.frequency, ps.frequency < item[1])
+            axis.plot(ps.frequency[to_plot], ps.power[to_plot], 'r', label=f'peak {idx}')
+        axis.legend()
+        axis.set_title(title)
+        axis.set_xlabel('Frequency [Hz]')
+    ax[1].set_xlim(frequency_exclusions[0][0] - 1.0, frequency_exclusions[-1][1] + 1.0)
+    ax[1].set_xscale('linear')
+    ax[0].set_ylabel('Power [V$^2$/Hz]')
+    plt.suptitle('Identified peaks')
+    plt.show()
+
```

Comment:
I think you forgot to include the figure `identify_peaks.png` here

Reply:
Indeed!